### ISSUE TITLE
drag+drop in frontend

### ISSUE THREAD

**Issue by Xiphe** [serverless/scope#8]
This is a conceptual question.

Do you see this project going into a direction of waffle.io or github projects in terms of write access in the frontend? Or do you want to keep it as a read-only dashboard?

**Comment by DavidWells** [comment]
Drag and drop would be cool but would require an auth layer on top of the UI and then we would need to map back the drag/drop actions to update issues/PR labels back into github.

It might make things complex and wasn't something we needed for https://serverless.com/framework/status

**Comment by Xiphe** [comment]
Yeah, I totally understand that. Just asking because I'm thinking about spending a few hours on this topic and see if I can get s.th. done.

**Comment by QuantumInformation** [comment]
@Xiphe I'm sure a fork would be a great example if you have the time.
